recipe/meta.yaml

```diff
@@ -20,6 +22,7 @@ requirements:
     - make  # [unix]
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
+    - sysroot_linux-64 2.17  # [linux64]
```

Comment:
Shall we try to undo this?

Reply:
Hm. Maybe. Perhaps it's not needed, right?

Reply:
Yes. I think your dl patch is the correct one.